kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/operator/KroxyliciousOperatorBundleInstaller.java

```diff
@@ -139,6 +121,13 @@ private void applyDeploymentFile() {
                 .get(0)
                 .getImage();
 
+        Map<String, Quantity> limits = operatorDeployment.getSpec().getTemplate().getSpec().getContainers().get(0).getResources().getLimits();
```

Comment:
curious - why do you need to do this?

Reply:
I'm really confused by your reply.  It is true you don't want more than one replica of the operator to run at once, but changing the cpu is not the right way to achieve it.

Reply:
Yeah, I will remove it once we agreed to wait for the Proxy scalation once it is available.